dnf-json

```diff
@@ -104,97 +108,103 @@ def repo_checksums(base):
     return checksums
 
 
-call = json.load(sys.stdin)
-command = call["command"]
-arguments = call["arguments"]
-repos = arguments.get("repos", {})
-arch = arguments["arch"]
-cachedir = arguments["cachedir"]
-module_platform_id = arguments["module_platform_id"]
-
-with tempfile.TemporaryDirectory() as persistdir:
-    try:
-        base = create_base(
-            repos,
-            module_platform_id,
-            persistdir,
-            cachedir,
-            arch
-        )
-    except dnf.exceptions.Error as e:
-        exit_with_dnf_error(
-            type(e).__name__,
```

Comment:
Out of curiosity, is `type(e).__name__` equal to `dnf.exceptions.Error` or just `Error` ?

Reply:
I believe that the name of the exception is going to be the name of the actual instantiated class.
For this particular feature, It would have gave the same behaviour with only `except Exception as e:`